Source/JavaScriptCore/wasm/WasmBBQJIT.cpp

```diff
@@ -5202,7 +5215,7 @@ class BBQJIT {
                     if (returnLocation.isGPR())
                         m_gprSet.add(returnLocation.asGPR(), Width::Width64);
                     else
-                        m_fprSet.add(returnLocation.asFPR(), Width::Width64);
+                        m_fprSet.add(returnLocation.asFPR(), Width::Width128);
```

Comment:
IgnoreVectors (so 32 bit arm works too) instead of an explicit Width64/128

Reply:
The new baseline is disabled on 32-bit ARM currently, so this is unnecessary for now. Let's revisit this though, I want to be clear on exactly when we need to be explicit and when `IgnoreVectors` is actually the right move.